Quorum/apis/llms/llm_conversation.py

```diff
@@ -0,0 +1,105 @@
+from enum import StrEnum, auto
+from pathlib import Path
+
+class MessageRole(StrEnum):
+    USER = auto()
+    ASSISTANT = auto()
+
+
+MAX_TOKENS = 1024
+
+
+def tmp_send_message_to_claude(client, model, messages):
+    try:
+        response = client.messages.create(
+            max_tokens=MAX_TOKENS,
+            model=model,
+            messages=messages
+        )
+    except:
+        raise
+
```

Comment:
Explicit error print with pp module would be nice to emphasis the root cause failure here.

Reply:
This whole function is temporary and I will probably remove it before merging this PR. I made sure to make it as ugly as possible for this reason.